charts/internal/machine-controller-manager/seed/templates/vpa.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     updateMode: {{ .Values.vpa.updatePolicy.updateMode | quote }}
   resourcePolicy:
     containerPolicies:
-    - containerName: machine-controller-manager-provider-openstack
+    - containerName: provider-openstack
```

Comment:
Sorry for being late to the party, I'm only seeing this now. That's a great catch, I suppose the reason why this happened is that this container has a different name in the openstack extension than in the other extensions. Therefore, can we fix it the other way round, for consistency reasons with the other extensions?
* aws extension container is named [`machine-controller-manager-provider-aws`](https://github.com/gardener/gardener-extension-provider-aws/blob/master/charts/internal/machine-controller-manager/seed/templates/deployment.yaml#L42)
* gcp extension container is named [`machine-controller-manager-provider-gcp`](https://github.com/gardener/gardener-extension-provider-gcp/blob/25adbe3828261d1ef1ffc9ad9b32ac90679157de/charts/internal/machine-controller-manager/seed/templates/deployment.yaml#L42)
* azure extension is named [`machine-controller-manager-provider-azure`](https://github.com/gardener/gardener-extension-provider-azure/blob/master/charts/internal/machine-controller-manager/seed/templates/deployment.yaml#L42)

Thanks!

Reply:
@voelzmo Sure. Do you want me to rename the container in the Deployment to `machine-controller-manager-provider-openstack` ?